

<?xml version="1.0" encoding="ISO-8859-15"?>
<launch>
    <arg name="node_start_delay" default="10.0" />
    <rosparam command="load" file="$(find voxblox_ros)/cfg/anymal.yml" />

    <node name="voxblox_tsdf_node" pkg="voxblox_ros" type="tsdf_server" output="screen" args="-alsologtostderr" clear_params="true" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' ">
        <remap from="pointcloud" to="/bpearl_front_back_todo" />
        <param name="world_frame" value="map" />
    </node>

    <node name="voxblox_height_node" pkg="voxblox_ros" type="height_server" output="screen" args="-alsologtostderr" clear_params="true" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' ">
        <remap from="pointcloud" to="/bpearl_front_back_todo" />
        <param name="world_frame" value="map" />
    </node>

</launch> 
